sdk/core/core-http/src/policies/deserializationPolicy.ts

```diff
@@ -270,7 +270,7 @@ function parse(
   xmlContentTypes: string[],
   operationResponse: HttpOperationResponse
 ): Promise<HttpOperationResponse> {
-  const errorHandler = (err: Error & { code: string }) => {
+  const errorHandler = (err: Error & { code: string }): Promise<any> => {
```

Comment:
should this be `Promise<never>` since it rejects with error?

Reply:
Thank you, Jeremy! This is my first time using never! :0